carlymclift/RancidTomatillos
# Jkw favorite movies

#### What's this PR do?
- Adds favorite capability with `http://localhost:3001/api/v1/favorites`
- Adds favorite icons to `Movie Page` & `Movie Card`, which are styled based on if the card is favorited or not.
- Adds a NavLink for `http://localhost:3000/favorites` so a user can view their favorite movies
#### Where should the reviewer start?
- Check out these files in tandem
#### How should this be manually tested?
- Fetch remote branch & make sure you have your `rancid-tomatillos-api` running on `localhost:3001`; Open the app & favorite/unfavorite some movies.
#### What are the relevant tickets?
- Close #32 
#### Screenshots (if appropriate)

![jkw-addFavorites-1](https://user-images.githubusercontent.com/62263439/91664451-a3b45380-eaac-11ea-90b3-d4804b4962a7.gif)
![jkw-addFavorites-2](https://user-images.githubusercontent.com/62263439/91664460-a8790780-eaac-11ea-9595-92260cf7b795.gif)

#### Questions:
- Do we only want to make this feature available when someone is logged in?
